docs/guide.mdx

```diff
@@ -61,10 +61,10 @@ If an access token isn't provided, the following rate limits apply:
 The Wave does not support the use of sha256 digest in the image name, e.g. `ubuntu@sha256:3235...ce8f`, when using
 the augmentation process to extend container images.
```

Comment:
```suggestion
the augmentation process to extend container images.
This is because Wave augmentation affects the image sha256, because the image contents change.
```